CLIMBING.ipynb

```diff
@@ -30,77 +30,46 @@
     "import pandas as pd\n",
     "import matplotlib.pyplot as plt\n",
     "\n",
-    "def bouldering():\n",
-    "    climbers = pd.read_csv('bouldering_data.csv')\n",
-    "    df = climbers[['Age', 'Bouldering Grade', 'Years Climbing']]\n",
+    "# Retrieving data from CSV files\n",
```

Comment:
Another small improvement that makes sense when writing Jupyter Notebooks is to run things in different cells.
This way you only run the import commands once at the beginning.
Then you run the file reading commands once at the beginning.

if you change the color of the widget you do not want to run everything from the top. You would only run the widget cell.